zebedee-cms/src/main/java/com/github/onsdigital/zebedee/util/CustomListCollector.java

```diff
@@ -25,7 +25,11 @@
 
     @Override
     public BiConsumer<ImmutableList.Builder<T>, T> accumulator() {
-        return (builder, item) -> builder.add(item);
+        return (builder, item) -> {
+            if (item != null) {
+                builder.add(item);
+            }
+        };
```

Comment:
This class allows you to use streams with UserList and TeamsList so its not used in changes iv made here but in the code that uses the collector - I did this change a while ago but I added the null check because it fell over when I made this change.